### ISSUE TITLE
Accessibility bug: Button label is incorrect in the move up and down buttons

### ISSUE THREAD

**Issue by enejb** [wordpress-mobile/gutenberg-mobile#2396]
**Describe the bug**
Currently when you are in the accessibility move using VoiceOver the text that is said to you don't reflect 

**To Reproduce**
1. Go to the editor. Write some content. 
2. Enable VoiceOver in iOS.
3. Navigate to the bottom or the very top move buttons. Notice what is being said is not what you expect to hear. 

watch:
https://cloudup.com/cmsx-nEVpoQ

**Expected behavior**
That when you are at the top the button should tell you to move the box to the bottom. 

When you are at the bottom you would expect that the button would tell you that you are moving the block up instead of down. 

**Screenshots**
https://cloudup.com/cmsx-nEVpoQ

**Smartphone (please complete the following information):**
 - Device: iPhone 11
 - OS: Latest iOS

**Additional context**
This was discover while accessibility testing. 

I think it would be good to refactor our code to have the same logic as core. :)

**Comment by etoledom** [comment]
Thank you for your report @enejb !

I'm testing it and can't reproduce the error.

Testing on the example project:
- Remove all content.
- Create two blocks.
- Enable VoiceOver.
- Go over the block mover buttons.
- VoiceOver tells the expected words.

Also tested on WPiOS.

Maybe there's something I'm missing?

**Comment by jd-alexander** [comment]
Same @etoledom I was not able to reproduce this either. Tested on WPiOS.

**Comment by enejb** [comment]
I just retested this and it seems to work as expected now. 
I am going to close this issue.
